src/Row.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from "react";
+
+const Row = props => {
+  return (
+    <tr
+      onClick={() => props.setSelectedRow(props.row.id)}
+      className={props.isSelected ? "selected-row" : "unselected-row"}
```

Comment:
This is **really** close to the correct solution, but not quite! The exercise asks you to let rows be highlighted *independently*. This means that  multiple rows can highlighted at once.

Your solution doesn't do that, but with some small changes it would. Can you figure out how to change it?